docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -79,17 +80,18 @@ You can also specify:
 
 - .. include:: ../reusables/threads-query-execution.rst
 
-
-.. pull-quote::
+. pull-quote::
```

Comment:
It looks as if we've accidentally lost a period here, resulting in bad formatting:

<img width="755" alt="image" src="https://user-images.githubusercontent.com/1877141/188015337-27a53723-8196-4919-97cf-3a95185547f5.png">

```suggestion
.. pull-quote::
```

Reply:
Thanks. Yes.